app/controllers/admin/group_users_controller.rb

```diff
@@ -3,6 +3,10 @@ class GroupUsersController < AdminController
     before_action :load_group
 
     def index
+      add_breadcrumb t(:'hyrax.controls.home'), root_path
+      add_breadcrumb t(:'hyrax.toolbar.admin.menu'), hyrax.admin_path
+      add_breadcrumb t(:'hyku.admin.groups.title.edit'), edit_admin_group_path(@group.id)
```

Comment:
You can just pass `@group` to `edit_admin_group_path(@group)` it knows how to find an id.